File: LsHcore/src/main/pl/edu/osp/tiles/CoolWater.java
package pl.edu.osp.tiles;

import pl.edu.osp.objects.Plankton;

public class CoolWater  extends Water {

		final private byte aType = 62;
		final protected byte fertility = 125;
		
		public CoolWater() {
			staticObject = new Plankton();
	}

}

File: lshcore/src/main/scala/eu/brosbit/lshcore/tiles/CoolWater.scala
package eu.brosbit.lshcore.tiles

import eu.brosbit.lshcore.immovable.*

object CoolWater extends TileObj:
  override val level: Byte = 0 
  override val fertility: Byte = 0
  override val name: String = "cool water"
  override val shortName: String = "wc" 

class CoolWater extends Tile:
  override val aType = CoolWater
